client/state/data-layer/wpcom/jetpack-install/index.js

```diff
@@ -40,15 +35,26 @@ export const handleResponse = ( { url }, data ) => {
 	if ( data.status ) {
 		return logToTracks( jetpackRemoteInstallComplete( url ) );
 	}
-	return logToTracks( jetpackRemoteInstallUpdateError( url, data.error.code, data.error.message ) );
+	return logToTracks( jetpackRemoteInstallUpdateError( url, 'UNKNOWN_ERROR' ) );
```

Comment:
> On reflection, I don't think this check serves any purpose now. Any non-error return from the API should be treated as success.

...meaning we can remove this call.

Reply:
Seems to me this is the case. I'd say it differently though: any error we did not identify and treated as such, will be labeled a success.